modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,35 @@
+= View Runtime Fabric Status and Resources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use Anypoint Runtime Manager to monitor the resource usage and health of your Anypoint Runtime Fabric clusters.
+
+== Resource Usage 
+
+To view the resource and vCPU usage:
+
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Click in the area between a Runtime Fabric name and its status to display resource and status information for that Runtime Fabric.
+. Click the *Manage* button to open the management page.
++
+Information that is displayed includes version, number of controller, worker, and ingress (internal load balancer) nodes, and  aggregated vCPU core usage.
```

Comment:
```suggestion
Information that is displayed includes version; number of controller, worker, and ingress (internal load balancer) nodes; and aggregated vCPU core usage.
```

Reply:
This is incorrect -- this shows the amount of vCPU the cluster has in total to use. It does **not** show vCPU used or available.